File: Cap 09/Topico 9.2/Topico 9.2.1 a 9.2.3 juntos/regsimples.py
'''
 * Livro .....: Algoritmos
 *              Logica para Desenvolvimento de Programacao de
 *              Computadores
 * Autores ...: Jose Augusto N. G. Manzano
 *              Jayr Figueiredo de Oliveira
 * Editora ...: Erica
 *
 * Capitulo ..: 9 - Topicos 9.2.1 a 9.2.3
 * 
 * Programa ..: MANIPULACAO SIMPLES DE REGISTRO
'''

class CAD_ALUNO:
  def __init__(self):
    self.NOME = ""
    self.TURMA = ""
    self.SALA = 0
    self.NOTA1 = 0.0
    self.NOTA2 = 0.0
    self.NOTA3 = 0.0
    self.NOTA4 = 0.0

ALUNO = CAD_ALUNO()

print("REGISTRO ESCOLAR - SIMPLES (1 ALUNO)\n\n")
ALUNO.NOME = input("Entre o nome ......: ")
ALUNO.TURMA = input("Entre a turma .....: ")
ALUNO.SALA = int(input("Entre a sala ......: "))
ALUNO.NOTA1 = float(input("Entre a 1a. nota ..: "))
ALUNO.NOTA2 = float(input("Entre a 2a. nota ..: "))
ALUNO.NOTA3 = float(input("Entre a 3a. nota ..: "))
ALUNO.NOTA4 = float(input("Entre a 4a. nota ..: "))

print()
print("DADOS DO ALUNO")
print("Nome ..............: ", ALUNO.NOME)
print("Turma .............: ", ALUNO.TURMA)
print("Sala ..............: ", "{:>4}".format(ALUNO.SALA))
print("Nota 1 ............: ", "{:>4.1f}".format(ALUNO.NOTA1))
print("Nota 2 ............: ", "{:>4.1f}".format(ALUNO.NOTA2))
print("Nota 3 ............: ", "{:>4.1f}".format(ALUNO.NOTA3))
print("Nota 4 ............: ", "{:>4.1f}".format(ALUNO.NOTA4))

File: Cap 09/Topico 9.2/Topico 9.2.1 a 9.2.3 juntos/regsimples.lua
--[[
 * Livro .....: Algoritmos
 *              Logica para Desenvolvimento de Programacao de
 *              Computadores
 * Autores ...: Jose Augusto N. G. Manzano
 *              Jayr Figueiredo de Oliveira
 * Editora ...: Erica
 *
 * Capitulo ..: 9 - Topicos 9.2.1 a 9.2.3
 * 
 * Programa ..: MANIPULACAO SIMPLES DE REGISTRO
--]]

CAD_ALUNO = {
  NOME = "",
  TURMA = "",
  SALA = 0,
  NOTA1 = 0.0,
  NOTA2 = 0.0,
  NOTA3 = 0.0,
  NOTA4 = 0.0
}

print("REGISTRO ESCOLAR - SIMPLES (1 ALUNO)\n")
io.write("Entre o nome ......: ")
CAD_ALUNO.NOME = io.read()
io.write("Entre a turma .....: ")
CAD_ALUNO.TURMA = io.read()
io.write("Entre a sala ......: ")
CAD_ALUNO.SALA = tonumber(io.read())
io.write("Entre a 1a. nota ..: ")
CAD_ALUNO.NOTA1 = tonumber(io.read())
io.write("Entre a 2a. nota ..: ")
CAD_ALUNO.NOTA2 = tonumber(io.read())
io.write("Entre a 3a. nota ..: ")
CAD_ALUNO.NOTA3 = tonumber(io.read())
io.write("Entre a 4a. nota ..: ")
CAD_ALUNO.NOTA4 = tonumber(io.read())

print()
print("DADOS DO ALUNO")
print("Nome ..............: " .. CAD_ALUNO.NOME)
print("Turma .............: " .. CAD_ALUNO.TURMA)
print("Sala ..............: " .. string.format("%4d", CAD_ALUNO.SALA))
print("Nota 1 ............: " .. string.format("%4.1f", CAD_ALUNO.NOTA1))
print("Nota 2 ............: " .. string.format("%4.1f", CAD_ALUNO.NOTA2))
print("Nota 3 ............: " .. string.format("%4.1f", CAD_ALUNO.NOTA3))
print("Nota 4 ............: " .. string.format("%4.1f", CAD_ALUNO.NOTA4))
